packages/pwa-kit-dev/src/utils/resolver-utils.js

```diff
@@ -80,44 +86,43 @@ export const expand = (extensions = []) =>
  * @param {String} sourcePath - The path the the file of the source import.
  * @param {Object} opts - The path the the file of the source import.
  * @param {Array<shortName: String, config: Array>} opts.extensions - List of extensions used by the base PWA-Kit application.
- * @param {String>} opts.productDir - Absolute path of the base project.
+ * @param {String>} opts.projectDir - Absolute path of the base project.
  * @returns {String[]} paths - The potential paths to find the module import.
  */
 export const buildCandidatePaths = (importPath, sourcePath, opts = {}) => {
     // Replace wildcard character as it has done its job getting us to this point.
     importPath = importPath.replace('*/', '')
 
-    const {extensions = [], productDir = process.cwd()} = opts
+    const {extensions = [], projectDir = process.cwd()} = opts
     const isSelfReferenceImport = isSelfReference(importPath, sourcePath)
```

Comment:
Is `isSelfReference` handling correctly full Windows paths?
E.g.
`'C:\\project\\node_modules\\module-extension\\src\\overrides\\app\\routes.jsx'`